code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -68,21 +68,21 @@
 	msg += "<EM>[src.name]</EM>!\n"
 
 	//uniform
-	if(w_uniform && !(slot_w_uniform in obscured))
+	if(w_uniform && !(slot_w_uniform in obscured) && !(w_uniform.invisibility || w_uniform.alpha <= 1))
```

Comment:
I don't like this copy-paste. Add a proc to do it, like `w_uniform.can_see_equipped()`.